src/openzaak/components/zaken/api/serializers/zaakobjecten.py

```diff
@@ -91,42 +91,46 @@ class ObjectTypeOverigeDefinitieSerializer(serializers.Serializer):
     )
 
 
+# TODO fix in cgac
+mapping = {
```

Comment:
for Rol and Zaakobject i had to add the discriminator explicitly, otherwise it does not exist [here](https://github.com/maykinmedia/commonground-api-common/blob/main/vng_api_common/polymorphism.py#L171) in attrs

The zaakobject discriminator has another issue because it uses group_field, which is used to alter the fields in mapping so that everything is nested, When ZaakObjectSubSerializer is loaded it does it again so it becomes object_informatie__object_informatie__...

currently fixed it by copying the mapping to both separately, another solution would maybe be to not add the group_field in the  ZaakObjectSubSerializer or change something in cgac.

Reply:
Could you add a todo for this and make an issue to fix this properly?